### PR TITLE
added possibility to sort saved wordpairs

### PR DESCRIPTION
**Pull request by nyko27**
Added button which allows user sort word pairs alphabetically that he previously marked as saved.

### CODE REVIEW

**Review by Markot96 — APPROVED** [review]
Great changes! You can merge

**Review by MarchukRoman063 — CHANGES_REQUESTED** [review]
Remove code duplication in map function

File: lib/main.dart [hunk 1]
```diff
@@ -68,33 +69,68 @@ class _RandomWordsState extends State<RandomWords> {
...
-                title: Text(
-                  pair.asPascalCase,
-                  style: _biggerFont,
-                ),
-              );
-            },
-          );
+          final tiles = _sorted.isNotEmpty
+              ? _sorted.map(
+                  (WordPair pair) {
```
Comment (igor-leshkevych) [inline lib/main.dart]: use lambda funcation to save your selector to a separate variable, and then reuse it in both clauses:
```
final tiles = _sorted.isNotEmpty ? _sorted.map(selector) : _saved.map(selector)
```

**Review by igor-leshkevych — COMMENTED** [review]
> Remove code duplication in map function

nice catch, but when your comment is related to a particular block of code, it's better to use an inline comment (so it is attached to that block)

**Comment by igor-leshkevych** [comment]
@nyko27 please attach some screenshots

**Comment by nyko27** [comment]
> please attach some screenshots

@igor-leshkevych 



Saved word pairs before being sorted.

image




Saved word pairs after user pressed button(top right corner) "Sort saved".

image

**Comment by igor-leshkevych** [comment]
@nyko27 

it's better to edit PR description and insert screenshots there rather than adding a new message to conversation. 

+5/5 hello world
+3/3 custom feature + PR

**Pull request merged**